nncf/experimental/post_training/model_transformer_handler.py

```diff
@@ -0,0 +1,35 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from xml.dom import NotFoundErr
+
+
+class ModelTransformersHandler:
```

Comment:
Might want to use the general-purpose `nncf.common.utils.registry.Registry` instead for consistency.

Reply:
Done